index.html

```diff
@@ -19,8 +19,176 @@ <h1>Sebastián<br>Muñoz</h1>
     <section class="intro">
         <p>Hello! I am a software developer! I can help you build a product, feature or website.      Taka a look of my works. <br> If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
     </section>
-    <div class="collab">
+    <div class="btn">
         <button type="button">Start collaboration</button>
     </div>
+    <div class="geometry">
+        <img src="assets/img/image_geometry_2.png" alt="">
+    </div>
+    <section class="projects-title">
+        <h1>Projects</h1>
+        <ul class="projects">
+            <li>
+                <div class="grid-container">
```

Comment:
- [ ] Looks like `grid` is used in the wrong place. I suggest use `grid` for  the projects class.